.github/workflows/deploy.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       BW_NUMBER_PROVIDER: ${{ secrets.BW_NUMBER_PROVIDER }}
     steps:
       - name: Set Release Version
-        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/v}" >> $GITHUB_ENV
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/tags/v}" >> $GITHUB_ENV
```

Comment:
Wouldn't this grab all version tags available? How do we just grab the latest one?